invenio.cfg

```diff
@@ -211,8 +223,12 @@ OAUTHCLIENT_CERN_USER_INFO_FROM_ENDPOINT = False
 
 handlers = _keycloak_helper.get_handlers()
 handlers["signup_handler"]["info_serializer"] = cern_info_serializer
-rest_handlers = _keycloak_helper.get_handlers()
-rest_handlers["signup_handler"]["info_serializer"] = cern_info_serializer
+handlers["signup_handler"]["groups_serializer"] = cern_group_serializer
```

Comment:
nit for readability:

```python
handlers["signup_handler"] = {
    info_serializer: ...
    groups_serializer: ...
    ...
}
```

carefully with the fact that dictionaries are references (not copies), but you can even reuse the dict above for both rest and ui handlers. But copy might actually be ugly, so the above for readability.